std_msgs/Header header

# oculii radar driver status msgs

string Radar_Alive
string Frame_Size
uint8 Frame_Rate
string GPS
uint8 nTracks

